tests/e2e/testdata/scenarios/direct/llm_immutable_field/labels/_object01.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: logging.cnrm.cloud.google.com/v1beta1
+kind: LoggingLogMetric
+metadata:
+  annotations:
+    cnrm.cloud.google.com/management-conflict-prevention-policy: none
+    cnrm.cloud.google.com/project-id: ${projectId}
+    cnrm.cloud.google.com/state-into-spec: absent
+  finalizers:
+  - cnrm.cloud.google.com/finalizer
+  - cnrm.cloud.google.com/deletion-defender
+  generation: 3
+  name: logginglogmetric-${uniqueId}
+  namespace: ${projectId}
+spec:
+  bucketOptions:
+    explicitBuckets:
+      bounds:
+      - 1.5
+      - 4.5
+  description: an updated description
+  filter: resource.type=gae_app AND severity<=ERROR
+  labelExtractors:
+    testkey: EXTRACT(jsonPayload.request)
+  metricDescriptor:
+    displayName: a concise name
+    labels:
+    - description: a label description
+      key: testkey
+      valueType: INT64
+    launchStage: EARLY_ACCESS
+    metadata:
+      ingestDelay: 1s
+      samplePeriod: 1s
+    metricKind: DELTA
+    valueType: DISTRIBUTION
+  projectRef:
+    external: projects/${projectId}
+  resourceID: logginglogmetric-${uniqueId}
+  valueExtractor: EXTRACT(jsonPayload.response)
+status:
+  conditions:
+  - lastTransitionTime: "1970-01-01T00:00:00Z"
+    message: |-
+      Update call failed: error updating: logMetric update failed: googleapi: Error 400: Cannot change value type of label: testkey
```

Comment:
Does this mean that the immutable webhook was not catching this change before?

Reply:
Yes, current webhook for DCL controller does not handle immutability on `spec.metricDescriptor.labels` fields(including `labels.description`,`labels. key`, `labels.valueType`). With DCL controller, we'll just let the GCP API to fail.

I also replied in this [comment](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1750#discussion_r1600925434).